iroh-resolver/src/dns_resolver.rs

```diff
@@ -0,0 +1,156 @@
+use crate::resolver::Path;
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::net::{IpAddr, Ipv4Addr};
+use trust_dns_resolver::config::{NameServerConfigGroup, ResolverConfig, ResolverOpts};
+use trust_dns_resolver::{AsyncResolver, TokioAsyncResolver};
+
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize)]
+pub struct DnsResolverConfig {
```

Comment:
This should be just `Config`, reads nicer and is more in line with conventions we use. That way imports look like `dns_resolver::Config`.

Reply:
Done